frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -95,25 +107,29 @@ export const CreateOCSServiceForm = withHandlePromise<
     match: {
       params: { appName, ns },
     },
+    csv,
   } = props;
   const [selectedNodes, setSelectedNodes] = React.useState<NodeKind[]>(null);
   const [visibleRows, setVisibleRows] = React.useState<NodeKind[]>(null);
   const [osdSize, setOSDSize] = React.useState(defaultRequestSize.NON_BAREMETAL);
   const [storageClass, setStorageClass] = React.useState<StorageClassResourceKind>(null);
   const dispatch = useDispatch();
+  const storageClusterExample = exampleForModel(csv, OCSServiceModel);
 
   const submit = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault();
     // eslint-disable-next-line promise/catch-or-return
-    handlePromise(makeOCSRequest(selectedNodes, storageClass, osdSize)).then(() => {
-      dispatch(setFlag(OCS_CONVERGED_FLAG, true));
-      dispatch(setFlag(OCS_FLAG, true));
-      history.push(
-        `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
-          OCSServiceModel,
-        )}/${getName(ocsRequestData)}`,
-      );
-    });
+    handlePromise(makeOCSRequest(storageClusterExample, selectedNodes, storageClass, osdSize)).then(
+      () => {
+        dispatch(setFlag(OCS_CONVERGED_FLAG, true));
+        dispatch(setFlag(OCS_FLAG, true));
+        history.push(
+          `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
+            OCSServiceModel,
+          )}/${OCS_CONVERGED_CR_NAME}`,
+        );
```

Comment:
No need to change this.

Reply:
I just add `storageClusterExample` to makeOCSRequest. and used `OCS_CONVERGED_CR_NAME`.